features/mbedtls/targets/TARGET_STM/TARGET_STM32F4/TARGET_NUCLEO_F439ZI/mbedtls_device.h

```diff
@@ -22,5 +22,7 @@
 
 #define MBEDTLS_AES_ALT
 
+#define MBEDTLS_MD5_ALT
 
+#define MBEDTLS_MD5_C
```

Comment:
This is an application defined configuration, and should not be defined in mbedtls_device.h